pkg/machines/components/vm/overview/cpuTypeModal.jsx

```diff
@@ -56,7 +61,7 @@ export const CPUTypeModal = ({ vm, models, close }) => {
                footer={
                    <>
                        {error && error.dialogError && <ModalError dialogError={error.dialogError} dialogErrorDetail={error.dialogErrorDetail} />}
-                       <Button variant='primary' id="cpu-config-dialog-apply" onClick={save}>
+                       <Button variant='primary' isDisabled={isLoading} isLoading={isLoading} id="cpu-config-dialog-apply" onClick={save}>
```

Comment:
What does the `isLoading` attribute do? The test does not use that, it should be happy with disabling the button while loading. Just a leftover?

Reply:
isLoading, adds a spinner to the button https://www.patternfly.org/v4/components/button/#progress 

The test does not do that, because that API call is instant normally, would be impossible to capture that.

Reply:
In other words, not a big deal more like to increase consistency with other modals.

Reply:
I see, so this is actually legit UI. Thanks!